### PR TITLE
kubeadm: remove todo about retry function change

### PR DESCRIPTION
**Pull request by aravindb26**
#### What type of PR is this?
I have evaluated TODO retry remove if feasible

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->
```release-note


```docs

```

### CODE REVIEW

**Comment by neolit123** [comment]
> EasyCLA — Missing CLA Authorization.

hi, you need to sign the CLA before this PR can be reviewed.

File: cmd/kubeadm/app/util/apiclient/idempotency.go [hunk 1]
```diff
@@ -339,10 +339,7 @@ func PatchNode(client clientset.Interface, nodeName string, patchFn func(*v1.Nod
 	return lastError
 }
 
-// GetConfigMapWithRetry tries to retrieve a ConfigMap using the given client,
-// retrying if we get an unexpected error.
-//
-// TODO: evaluate if this can be done better. Potentially remove the retry if feasible.
```
Comment (neolit123) [inline cmd/kubeadm/app/util/apiclient/idempotency.go:345]: this also removes the go comment about the function.

removing the TODO seems OK to me, given it has stayed for a while and we did not change it.
related to https://github.com/kubernetes/kubeadm/issues/1606

Comment (aravindb26) [reply cmd/kubeadm/app/util/apiclient/idempotency.go:345]: /easycla

Comment (aravindb26) [reply cmd/kubeadm/app/util/apiclient/idempotency.go:345]: Yeah Only That one comment "TODO" needs to be removed, not the go comment too, it happened mistakenly.

And I have signed the CLA

**Review by aravindb26 — COMMENTED** [review]
Done

**Review by neolit123 — COMMENTED** [review]
please squash the commits to 1

**Comment by neolit123** [comment]
/retitle kubeadm: remove todo about retry function change

**Comment by aravindb26** [comment]
> please squash the commits to 1

yeah done

**Comment by neolit123** [comment]
godmt is failing. try running `./hack/verify-gofmt.sh` locally and make updates

**Comment by aravindb26** [comment]
> godmt is failing. try running `./hack/verify-gofmt.sh` locally and make updates

Who should run that?

**Comment by neolit123** [comment]
> > godmt is failing. try running `./hack/verify-gofmt.sh` locally and make updates
> 
> Who should run that?

you can run it locally on your PC.

**Comment by neolit123** [comment]
the gofmt check on the PR must pass or your PR will not merge.

**Comment by aravindb26** [comment]
> the gofmt check on the PR must pass or your PR will not merge.

Ok, but who should resolve these issues, your side or my side?

**Comment by aravindb26** [comment]
why it's failing? is it the issue from my side or your side?

**Review by aravindb26 — COMMENTED** [review]
/retest

File: cmd/kubeadm/app/util/apiclient/idempotency.go [hunk 2]
```diff
@@ -341,8 +341,7 @@ func PatchNode(client clientset.Interface, nodeName string, patchFn func(*v1.Nod
 
 // GetConfigMapWithRetry tries to retrieve a ConfigMap using the given client,
 // retrying if we get an unexpected error.
-//
-// TODO: evaluate if this can be done better. Potentially remove the retry if feasible.
+
```
Comment (neolit123) [inline cmd/kubeadm/app/util/apiclient/idempotency.go:344]: you can just remove the empty line?
this should work:

```
// comment
// comment
function 
```

**Review by aravindb26 — COMMENTED** [review]
deleted the spaces

**Comment by pacoxu** [comment]
/label tide/merge-method-squash 
Better to squash to 1 commit.
/lgtm

**Comment by aravindb26** [comment]
> /label tide/merge-method-squash Better to squash to 1 commit. /lgtm

yeah done

**Comment by aravindb26** [comment]
@pacoxu why it's failing again? I done all correct this time

**Comment by aravindb26** [comment]
@pacoxu can we consider these under a bug?

**Comment by pacoxu** [comment]
I am not sure. Probably a flake failure.

**Comment by aravindb26** [comment]
> I am not sure. Probably a flake failure.

I saw other few pull requests were under kind/bug tag. If it's considered as a bug I want to submit these report to Google according to their terms and conditions being mentioned by them.

**Comment by pacoxu** [comment]
> I saw other few pull requests were under kind/bug tag. If it's considered as a bug I want to submit these report to Google according to their terms and conditions being mentioned by them.

Do you mean the failure in https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/112383/pull-kubernetes-integration/1569878607948419072?

TestDSCUpdatesPodLabelAfterDedupCurHistories seems to be a flake. You may open an issue in https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Fflake&template=flaking-test.yaml for this temporarily failure as it passed in a retest job.

**Comment by aravindb26** [comment]
> > I saw other few pull requests were under kind/bug tag. If it's considered as a bug I want to submit these report to Google according to their terms and conditions being mentioned by them.
> 
> Do you mean the failure in https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/112383/pull-kubernetes-integration/1569878607948419072?
> 
> TestDSCUpdatesPodLabelAfterDedupCurHistories seems to be a flake. You may open an issue in https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Fflake&template=flaking-test.yaml for this temporary failure as it passed in a retest job.

I mean here itself you can add a tag as bug "kind/bug".

**Comment by pacoxu** [comment]
The change is only about comments. I think this is a cleanup.

See more in https://github.com/kubernetes/community/blob/9406b4352fe2d5810cb21cc3cb059ce5886de157/contributors/devel/sig-node/triage.md#needs-kind

**Comment by aravindb26** [comment]
> The change is only about comments. I think this is a cleanup.
> 
> See more in https://github.com/kubernetes/community/blob/9406b4352fe2d5810cb21cc3cb059ce5886de157/contributors/devel/sig-node/triage.md#needs-kind

Yeah it should be in "kind/bug" I read that

**Comment by aravindb26** [comment]
> The change is only about comments. I think this is a cleanup.
> 
> See more in https://github.com/kubernetes/community/blob/9406b4352fe2d5810cb21cc3cb059ce5886de157/contributors/devel/sig-node/triage.md#needs-kind

I want to report this issue here as Google mentioned to submit open source issues after the fix here: https://bughunters.google.com/report/vrp

**Pull request merged**